Extension/package.nls.json

```diff
@@ -213,7 +213,11 @@
     "c_cpp.debuggers.serverLaunchTimeout.description": "Optional time, in milliseconds, for the debugger to wait for the debugServer to start up. Default is 10000.",
     "c_cpp.debuggers.coreDumpPath.description": "Optional full path to a core dump file for the specified program. Defaults to null.",
     "c_cpp.debuggers.cppdbg.externalConsole.description": "If true, a console is launched for the debuggee. If false, on Linux and Windows, it will appear in the Integrated Console.",
-    "c_cpp.debuggers.cppvsdbg.externalConsole.description": "If true, a console is launched for the debuggee. If false, no console is launched.",
+    "c_cpp.debuggers.cppvsdbg.console.description": "Where to launch the debug target.",
+    "c_cpp.debuggers.cppvsdbg.console.internalConsole.description": "Output to the VS Code Debug Console. This doesn't support reading console input (ex:Console.ReadLine)",
+    "c_cpp.debuggers.cppvsdbg.console.integratedTerminal.description": "VS Code's integrated terminal",
+    "c_cpp.debuggers.cppvsdbg.console.externalTerminal.description": "External terminal that can be configured via user settings",
```

Comment:
The description for externalTerminal is wrong - we will always just launch VsDebugConsole.exe as a new top level window using CreateProcess.

Reply:
I would suggest: Console applications will be launched in an external terminal window. The window will be reused in relaunch scenarios, and will not automatically disappear when the application exits.

---
In reply to: [558714356](https://github.com/microsoft/vscode-cpptools/pull/6794#discussion_r558714356) [](ancestors = 558714356)